css/styles.css

```diff
@@ -0,0 +1,146 @@
+body {
+  background-color: #f0f0f0;
+  font-family: "Open Sans", sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full-header {
+  flex-basis: 100%;
+}
+
+.main-container {
+  display: flex;
+  max-width: 800px;
+  align-items: center;
+  margin: auto;
+  justify-items: center;
+  padding-left: 1rem;
+}
+
+.head-container {
+  height: 3.5rem;
+  margin: auto;
+  align-items: center;
+  background-color: #562b80;
+}
```

Comment:
Instead of making your own variants of the `container` class, you should use the `container` class provided in `layout.css`, and make additional classes for any styling you need beyond what the `container` class has applied to it